.github/workflows/test-workflow.yml

```diff
@@ -20,6 +20,14 @@ jobs:
       with:
         node-version: "${{ steps.nvm.outputs.NVMRC }}"
 
+    - name: Cache Dependencies
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-
+
     - name: Install Dependencies
       run: npm ci
```

Comment:
This workflow still runs a clean install from the lockfile, which means the caching does nothing and just slows down the workflow. This needs a check added for if the cache had a hit?

Reply:
Looks like if we're caching `~/.npm` we can just `npm ci` either way.